subprojects/packagefiles/glfw/meson.build

```diff
@@ -0,0 +1,259 @@
+#####  GLFW  #####
+#@ Project file
+project('glfw', 'c'
+       , version : '3.3.7'
+       , license : 'zlib'
+       , default_options : ['default_library=static'
+                           ,'b_ndebug=if-release'
+                           ,'c_std=c99'
+                           ,'warning_level=0'
+                           ,'werror=false']
+       , meson_version : '>=0.61.0')
+#@ Project variables
+sys_os = host_machine.system()               #  Returns the OS set in the native or cross configuration file
+sys_cc = meson.get_compiler('c')             #  Returns the C compiler set in the native or cross configuration file
+msn_ver = meson.version()                    #  Returns the version of Meson in use
+prj_xx = meson.is_cross_build()              #  Returns TRUE if the current build is a cross
+prj_sub = meson.is_subproject()              #  Returns TRUE if the current build is a subproject
+#
+opt_prefix = get_option('prefix')            #  Returns the install path prefix
+opt_bindir = get_option('bindir')            #  Returns the binary subdirectory
+opt_libdir = get_option('libdir')            #  Returns the library subdirectory
+opt_incdir = get_option('includedir')        #  Returns the include subdirectory
+opt_datadir = get_option('datadir')          #  Returns the data subdirectory
+#
+opt_docs = get_option('build-docs')          #  Build documentation? T/F
+opt_tests = get_option('build-tests')        #  Build test programs? T/F
+opt_examples = get_option('build-examples')  #  Build example programs? T/F
+opt_install = get_option('install')          #  Install GLFW? T/F
+opt_display = get_option('display-api')      #  Returns the selected backend API
+#@ Extra variables
+if (sys_os == 'dragonfly' or sys_os.endswith('bsd'))  #
+  bsd = ['bsd']                                           #  Unifies the BSDs in one reference
+endif                                                     #
```

Comment:
At the time this is used, it may be undefined because it's only created in a conditional. Also why is it an array?

You may want to define an is_bsd variable instead.

Reply:
Oh! It's an array because at one point I had it as `['bsd', true]` depending on how I wanted to reference it, then realized that it was unnecessary.